downstream/modules/eda/con-event-streams.adoc

```diff
@@ -25,11 +25,11 @@ h| OAuth2 with JWT | Uses M2M mode with a grant type called *client credentials*
 
 h| ECDSA | Elliptic Curve Digital Signature Algorithm | SendGrid, Twilio
 
-h| Mutual TLS | Needs the vendor's CA certificate to be present in our servers at startup. This supports non-repudiation. 
-| PagerDuty
+//[Jameria] Not currently supported; will leave commented out for now in the event that it is supported in the near future.  h| Mutual TLS | Needs the vendor's CA certificate to be present in our servers at startup. This supports non-repudiation. 
+// | PagerDuty
 |===
 
-{EDAcontroller} also supports 4 other specialized event streams that are based on the 7 basic event stream types: 
+{EDAcontroller} also supports four other specialized event streams that are based on the seven basic event stream types: 
```

Comment:
they would be six after removing mutual TLS, right?

Reply:
Yes, that's correct. I've updated the amount. Thanks for catching that.

Reply:
I still see "seven" in this line, should not it be "six" ?

Reply:
Corrected.